_plugins/content_code.rb

```diff
@@ -0,0 +1,24 @@
+require 'rouge'
+
+# Include tabindex for accessibility reasons
```

Comment:
Might be helpful to point to the original implementation for future maintainer reference.

```suggestion
# Include tabindex for accessibility reasons
# See: https://github.com/rouge-ruby/rouge/blob/master/lib/rouge/formatters/html_pygments.rb
```